Sparkle/SUUpdater.m

```diff
@@ -253,13 +256,20 @@ - (void)updateDriverDidFinish:(NSNotification *)note
 
 - (NSDate *)lastUpdateCheckDate
 {
-    return [self.host objectForUserDefaultsKey:SULastCheckTimeKey];
+    if (![self updateLastCheckedDate])
+    {
+        [self setUpdateLastCheckedDate:[self.host objectForUserDefaultsKey:SULastCheckTimeKey]];
+    }
+    
+    return [self updateLastCheckedDate];
 }
 
 - (void)updateLastUpdateCheckDate
 {
+    [self setUpdateLastCheckedDate:[NSDate date]];
```

Comment:
I think this should be after willChangeValueForKey, so that before/after values in observers work